using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -111,32 +111,58 @@ If the provided URL is nil or empty, it is returned as is. Otherwise, the follow
 **Objective-C**
 
 ```objectivec
-NSURL* url = [[NSURL alloc] initWithString:@"www.myUrl.com"];
+NSURL* url = [[NSURL alloc] initWithString:@"https://example.com"];
 [ACPIdentity appendToUrl:url withCallback:^(NSURL * _Nullable urlWithVisitorData) {    
-// handle the appended url here}
+	// handle the appended url here
+	
+	// APIs which update the UI must be from main thread
```

Comment:
must be called from main thread - same comment for the next examples

Reply:
Fixed, thanks!